### PR TITLE
dev/build fixes

### PR DESCRIPTION
**Pull request by lekoala**
Checking if classes exists before request otherwise it might prevent dev/build from working properly

### CODE REVIEW

File: src/RequestFilter.php [hunk 1]
```diff
@@ -59,7 +60,7 @@ public function preRequest(SS_HTTPRequest $request, Session $session, DataModel
      * @return boolean Whether to continue processing other filters. Null or true will continue processing (optional)
      */
     public function postRequest(SS_HTTPRequest $request, SS_HTTPResponse $response, DataModel $model) {
-        if (Director::isDev()) {
+        if ($this->clockwork) {
```
Comment (markguinn) [inline src/RequestFilter.php]: Could we change this to `isset($this->clockwork)`?

**Comment by markguinn** [comment]
Thanks! This is a good idea. I suggested one change above. Would you have time to make that change and squash the 3 commits into 1? If not, I can squash them and merge manually and add the other change myself at a later time.

**Comment by lekoala** [comment]
done!

**Comment by markguinn** [comment]
Brilliant! Thanks.

**Pull request merged**